engine/src/test/java/com/redhat/cloud/notifications/TestHelpers.java

```diff
@@ -72,8 +73,7 @@ public static EmailAggregation createEmailAggregation(String orgId, String bundl
 
         emailActionMessage.setOrgId(orgId);
 
-        JsonObject payload = baseTransformer.toJsonObject(emailActionMessage);
-        aggregation.setPayload(payload);
+        aggregation.setPayload(TestHelpers.wrapActionToJsonObject(emailActionMessage));
```

Comment:
Is `TestHelpers#baseTransformer` still used?

Reply:
Nope, I removed it!